Makefile

```diff
@@ -51,6 +55,7 @@ install_fasm2bels:
 env:
 	echo ". `pwd`/third_party/rapidwright.sh" > "env.sh"
 	echo "export INTERCHANGE_SCHEMA_PATH=`pwd`/third_party/RapidWright/interchange/fpga-interchange-schema/interchange" >> "env.sh"
+	echo "export VIVADO_PATH=/tools/Xilinx/Vivado/2022.2/bin/vivado" >> "env.sh"
```

Comment:
This is also specified in config.py.  Do we need it both places?

Reply:
Good point.  The environment variable needs to exist for the fasm2bels build, so I'm just fixing that here.

But we could update our code to use this environment variable instead of the one in config.py.  Or update config.py to pull from the environment variable?  What do you think?

Reply:
I think that since the makefile is part of the setup, we can't assume the config.py is up to date, so config.py should pull from the environment.  How does that sound?

Reply:
Fixed.